content/en/community/hackathons/usoc23/overview/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Session 03 - Behind the Scenes
+linkTitle: Working with Unikraft
+weight: 5
+summary: |
+  We present the setup. The students will use the scripts to run and test all the applications available.
+  Expected time: 100 minutes
+--- 
+
+## Reminders
+
+### Setup
+
+{{< readfile file="/community/hackathons/sessions/setup/content/Setup.md" markdown="true" >}}
+
+### Support Files
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Required-Tools-and-Resources_Support-Files.md" markdown="true" >}}
+
+### linuxu and KVM
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_02.-linuxu-and-KVM.md" markdown="true" >}}
+
+## Overview
+
+### Unikraft Core
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_03.-Unikraft-Core.md" markdown="true" >}}
+
+### Configuring Unikraft - Config.uk
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_05.-Configuring-Unikraft---Config.uk.md" markdown="true" >}}
+
+### The Build System - basics
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_06.-The-Build-System---basics.md" markdown="true" >}}
+
+## Work items
+
+### 01. Tutorial / Reminder: Building and Running Unikraft
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-Tutorial---Reminder:-Building-and-Running-Unikraft.md" markdown="true" >}}
+
+#### Set Up
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-Set-Up.md" markdown="true" >}}
+
+#### linuxu, x86_64
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-linuxu-x86_64.md" markdown="true" >}}
+
+#### KVM, x86_64
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-KVM-x86_64.md" markdown="true" >}}
```

Comment:
Change file name to `_qemu-` instead of `_kvm-`

Reply:
`find an explanation to the differences.`
`find an explanation for the differences.`

Reply:
Hi @craciunoiuc,

I propose we change the name from qemu to kvm in another PR, because there will be changes in other sessions too and we should ensure that they are working too